### ISSUE TITLE
New Sermon does not save the selected 'service type'

### ISSUE THREAD

**Issue by s4069b** [WP-for-Church/Sermon-Manager#115]
I recently upgraded to version 2.7.2 on a Wordpress 4.8.2 install and now when I create a new sermon the 'service type' selected from the dropdown is not saved. This was repeatable when attempting to use another service type. 

My work-around at the moment is to go to 'all sermons' and 'quick edit' to manually insert the service type (which appears blank in quick-edit) and save. Then the service type is retained.

**Comment by nikola3244** [comment]
Hey,  we tried reproducing the issue, but Service Type gets saved each time.
Can you please tell us what browser, PHP and WordPress versions are you using?

Thanks

**Comment by nikola3244** [comment]
We managed to reproduce it. Note to self: Can not be rendered from `wp_inssert_post` action

**Comment by s4069b** [comment]
Sorry for my very very slow reply. I can't recall what browser I used last week - but reproduced using Google Chrome just now. Wordpress 4.8.2. and PHP 7.0.24. 

The sermon saves without throwing an error etc. The only hint I had of the error was that new sermons were not showing up when viewing a particular service-type. When I went into the list of 'all sermons' and looked at the recent ones under 'quick-edit' I noticed they had a blank 'service type'. When I added the service type manually under 'quick-edit' I was back in business. 

I see you have been able to reproduce the error yourself now.
